### PR TITLE
Review ready

### PR DESCRIPTION
**Pull request by ddddavis**
Fixed changes needed for review

### CODE REVIEW

**Review by fatymahmed — COMMENTED** [review]
## Hi @davisdambis @BrittanyBlake,
Great job making the changes suggested in the previous review 👏👏. However, there are still a few things that you need to fix.

- Your recommended videos sidebar overlaps with the main video, please fix this.
- You could add some spacing between the logo and the main icon.
screenshot
- Your search bar should have a `<button>` tag that represents the search button with has the search icon, [screenshot from Pathwright](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/embedding_images_and_video/search_button_requirement.png) 
- Your pull request should have a description of what you've done so far, what you have implemented, please make it more descriptive. [screenshot from Pathwright](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_prs.png)

[OPTIONAL]
- I noticed that this is your second review, in this case, it's a good idea to use the same pull request as the one you used for your first code review or you could add a link of the first pull request in this pull request. You can apply this to your upcoming projects.

**Review by fatymahmed — CHANGES_REQUESTED** [review]
### Kindly make the suggested changes and request for another code review.

## Status: Changes Required
_Feel free to reach out to me on slack (fatima) if you have any questions, I'd be happy to help._

**Comment by ddddavis** [comment]
- fixed sidebar and video overlap

- added spacing between icon and main logo

- added button tag

**Comment by shshamim63** [comment]
## Status :heavy_check_mark:  [Approved] :heavy_check_mark: 
___
Hi @davisdambis  and @BrittanyBlake :wave: 
Great job so far :clap: You have covered all the changes asked by the previous reviewer. Project contains descriptive readme but the appearance needs some polishing. Check this [link](https://www.markdownguide.org/cheat-sheet/). I think you can use it to look it better. The UI contains all the parts. maybe you can improve it a little bit more. Especially the nav items. The text of the aside items are really small comparing to its image. But as you have covered all the minimal requirements I will approve this project.
___
#### This :+1: is from me. You can go ahead. Don't forget to fill the project completion form.
___
## :herb: Happy Coding :herb: 
### Shakhawat Hossain
[Linkedin](https://www.linkedin.com/in/shakhawathossainshamim/)

File: index.html [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+        </section>
+        <section class="description">
+            <p id="description"> How do you contribute to open source? In this video, I'll describe the easiest way to make a simple contribution to an open source project -- without using the command line, a text editor, "forking a repo", or even touching Git outside of
+                Github. I'll also describe what kinds of contributions open source software projects are typically looking for. As an example, I use submitting a student solution to a web development exercise from The Odin Project, a free online curriculum
+                for learning web development with Ruby on Rails. You can find more information about that project at <a href="http://theodinproject.com.">http://theodinproject.com.</a></p>
+        </section>
+    </main>
+
+    <!--Sidebar-->
+    <section class="sidebar">
```
Comment (addod19) [inline index.html]: - Let improve the semantics by using ``` <aside>``` instead of ```<section class="sidebar">```

**Review by addod19 — CHANGES_REQUESTED** [review]
#### status: Require changes 🔴
Hi @davisdambis and @BrittanyBlake , 🙂
You have done a good job so far, there are few things we have to do to get this project approved check comment below : 🙂

- It has a <button> tag that represents the search button. The search button does not look good, let us improve it. [youtube.com](youtube.com)
- It has at least three images at the rightmost part of the navbar, which consist of icons and avatar. Currently you do not have the avatar on your page. Per theis requirement, it is assumed the user is logged in.
- It shows the avatar of the user who uploaded the image, currently we don't have this on our page. Let's add it.
- There is an horizontal scroll on the output of the page. Below is a screenshot of how it looks now:
Screenshot from 2020-01-17 19-11-09

This is due to giving a definite width to your main container. Let's give our main container a width of 100%.
- Let use fix the warnings return by [w3validator](https://validator.w3.org). They are warnings but easily fixable.
- Add descriptive pull request and summary.

Kindly make these changes and submit project for review 
Happy coding.

[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo-9738b0128/)

**Comment by ddddavis** [comment]
We improved the search button, changed section to aside and made changes to the project

**Comment by rOluochKe** [comment]
@addod19 and @davisdambis merge your project and resubmit for review

**Pull request merged**